bootstrapper-maven-plugin/src/main/resources/templates/Status.java

```diff
@@ -0,0 +1,7 @@
+package {{groupId}};
+
+import io.javaoperatorsdk.operator.api.ObservedGenerationAwareStatus;
+
+public class {{artifactClassId}}Status extends ObservedGenerationAwareStatus {
```

Comment:
why the extends clause?

Reply:
it is just to not have the status empty, we can have simpler status in theory, but in the sample observed generation makes sense.

Reply:
But I'm also ok with an empty status.

Reply:
It could actually make sense to make status classes handle observed generation automatically instead of being opt-in. If it's a best practice, maybe it should be enforced by default and made it opt-out? (I forgot we had that feature, to be honest 😅 )